Jenkinsfile-curriculum

```diff
@@ -0,0 +1,8 @@
+node {
+    stage('Load Jenkinsfiles') {
+        checkout scm
+    }
+
+    // TODO: Determine Jenkinsfile to run based on git diff
+    load 'adopt-a-pup/web-app/Jenkinsfile'
```

Comment:
Not sure how we are going to decide which Jenkinsfile to run when we have many of them. Ideas?